<launch>
<!-- Launch file for Livox MID360 LiDAR -->

	<arg name="rviz" default="true" />
	<arg name="bagfile" default="" />
	<arg name="save_dir" default="$(find fast_lio)/Log" />
	<arg name="odom_mode" default="0" />
	<rosparam command="load" file="$(find fast_lio)/config/mid360_pc2.yaml" />

    <param name="odom_mode" type="int" value="$(arg odom_mode)" />
	<param name="bagfile" type="string" value="$(arg bagfile)" />
	<param name="save_dir" type="string" value="$(arg save_dir)" />
    <node pkg="fast_lio" type="fastlio_mapping" name="laserMapping" output="screen" /> 

	<group if="$(arg rviz)">
	<node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find fast_lio)/rviz_cfg/loam_livox.rviz" />
	</group>

</launch>
